pulumi/grapl/__main__.py

```diff
@@ -544,6 +567,17 @@ def main() -> None:
             plugin_work_queue_postgres.password(),
         )
 
+        pulumi.export("uid-allocator-db-hostname", uid_allocator_postgres.host())
+        pulumi.export("uid-allocator-db-port", uid_allocator_postgres.port().apply(str))
+        pulumi.export(
+            "uid-allocator-db-username",
+            uid_allocator_postgres.username(),
+        )
+        pulumi.export(
+            "uid-allocator-db-password",
+            uid_allocator_postgres.password(),
+        )
+
```

Comment:
You don't need to export these unless they're used in integration tests (they're not)